packages/generator-electrode-component/app/index.js

```diff
@@ -136,6 +145,9 @@ var ReactComponentGenerator = yeoman.generators.Base.extend({
 	},
 
 	end: function() {
+		if (this.quoteType === "'") {
+			this.spawnCommandSync("node_modules/.bin/eslint", ["--fix", "src", "demo", "example", "test", "--ext", ".js,.jsx"]);
```

Comment:
right, I forgot that eslint can do this.  so no need to use prettier.

Is eslint added to the `dependencies` though?